### DISCUSSION TITLE
Practica CSS con flexbox y javascript

### DISCUSSION THREAD

**Discussion by OscarPedroza** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#388] [Show and tell]
Buenas tardes compañeros, me gustaría compartir mi práctica de CSS realizada con felxbox y Javascript para la funcionalidad, aún estoy realizando la parte del marketing.

Este es el link: [https://oscarpedroza.github.io/PracticaVaccination.github.io/](https://oscarpedroza.github.io/PracticaVaccination.github.io/)

A su vez les comparto esta librería de Javascript open source para realizar animaciones al hacer scroll en la página:
Libreria: [https://michalsnik.github.io/aos/](https://michalsnik.github.io/aos/)

**Comment by Josue-HernandezL** [comment]
disculpa, de donde sacaste los icons? que fuente tomaste
es que estoy usando unos pero me fallan, no se ponen

**Reply by Ingrzs** [reply]
Font awesome

**Reply by OscarPedroza** [reply]
Exacto utilice font awesome.

**Comment by JuanSalvador73** [comment]
Te quedaron increíbles esos FX en tu sitio, con tu permiso voy a bajarlos y estudiarlos. 🗡️ 
Gracias.

**Reply by OscarPedroza** [reply]
Muchas gracias compañero! no hay ningún problema, olvide incluir el enlace al repositorio, [https://github.com/OscarPedroza/PracticaVaccination.github.io](https://github.com/OscarPedroza/PracticaVaccination.github.io)
